tests/lib/test_openstack_hypervisor.py

```diff
@@ -1,10 +1,27 @@
 import unittest
 from unittest.mock import MagicMock
 
+from nose.tools import raises
+
 from openstack_api.openstack_hypervisor import OpenstackHypervisor
 
+from tests.lib.test_openstack_query_base import OpenstackQueryBaseTests
+
+
+# pylint:disable=too-few-public-methods
+class _HypervisorMock:
```

Comment:
This could be a dataclass AFAIK
As we can still override methods in DataClasses too